src/test/java/GmailPage.java

```diff
@@ -0,0 +1,243 @@
+    @FindBy(id = "profileIdentifier")
+    private WebElement dropDownWithEmailToExpand;
+    @FindBy(css = BUTTON_TO_DELETE_SELECTED_ACCOUNT)
+    private WebElement buttonToDeleteSelectedAccount;
+    @FindBy(css = EMAIL_TO_SELECT_AND_DELETE)
+    private WebElement emailToSelectAndDelete;
+    @FindBy(css = BUTTON_TO_CONFIRM_EMAIL_DELETION)
+    private WebElement buttonToConfirmEmailDeletion;
+    @FindBy(css = "div.T-I.J-J5-Ji.T-I-KE.L3")
+    private WebElement linkCreateNewEmail;
+    @FindBy(css = INPUT_EMAIL_TO)
+    private WebElement inputEmailTo;
+    @FindBy(css = "input[name = 'subjectbox']")
+    private WebElement inputEmailSubject;
+    @FindBy(css = "div[aria-label = 'Message Body']")
+    private WebElement inputEmailMessageBody;
+    @FindBy(css = "div[role = 'button'][aria-label *= 'Send']")
+    private WebElement buttonSendEmail;
+    @FindBy(css = "tbody tr[class = 'zA zE'], tr[class = 'zA yO']")
+    private List<WebElement> allRowsWithInboxEmails;
+    @FindBy(css = "div.ajl.aib.aZ6 div.TK div.TO[data-tooltip = 'Sent']")
+    private WebElement linkToSendFolder;
+    @FindBy(css = "input[name = 'q']")
+    private WebElement inputFilter;
+    @FindBy(css = "div.BltHke.nH.oy8Mbf[role = 'main'] tbody tr")
+    private List<WebElement> allRowsWithSentEmails;
+    @FindBy(css = "td.bq4.xY ul.bqY li:nth-child(2)")
+    private WebElement iconDeleteEmail;
+    @FindBy(css = "div.ajl.aib.aZ6 div.TK div.TO[data-tooltip = 'Trash']")
+    private WebElement linkToTrashFolderENG;
+    @FindBy(css = "div.ajl.aib.aZ6 div.TK div.TO[data-tooltip = 'Корзина']")
+    private WebElement linkToTrashFolderRUS;
+    @FindBy(css = "div.BltHke.nH.oy8Mbf[role = 'main'] tbody tr")
+    private List<WebElement> allRowsWithTrashEmails;
+    @FindBy(css = "div.G-tF div.Bn")
+    private WebElement buttonDeleteEmailForever;
+
+    public void waitUntilLinkToLoggedUserPresent() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector(LINK_TO_LOGGED_USER)));
+    }
+
+    public boolean loggedUserContainsLoginUsername(String loginUsername) {
+        return linkToLoggedUser.getAttribute(ATTRIBUTE_LOGGED_USER).contains(loginUsername);
+    }
+
+    public void waitUntilMaskPreventingClickDisappear() {
+        wait.until(ExpectedConditions.invisibilityOfElementLocated(By.cssSelector(MASK_PREVENTING_CLICK)));
+    }
+
+    public void waitUntilButtonToDelSelectedAcctPresent() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector(BUTTON_TO_DELETE_SELECTED_ACCOUNT)));
+    }
+
+    public void waitUntilEmailToSelectAndDeletePresent() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector(EMAIL_TO_SELECT_AND_DELETE)));
+    }
+
+    public void waitUntilButtonToConfirmEmailDeletionPresent() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector(BUTTON_TO_CONFIRM_EMAIL_DELETION)));
+    }
+
+    public LoginPage logout() {
```

Comment:
Do not afraid to add extra empty lines in code of methods. I add them between some logic interactions, f.e.:
        linkToLoggedUser.click();
        linkToSignOut.click();
        dropDownWithEmailToExpand.click();
        waitUntilMaskPreventingClickDisappear();
        waitUntilButtonToDelSelectedAcctPresent();

        buttonToDeleteSelectedAccount.click();
        waitUntilEmailToSelectAndDeletePresent();

        emailToSelectAndDelete.click();
        waitUntilButtonToConfirmEmailDeletionPresent();

        buttonToConfirmEmailDeletion.click();

        return new LoginPage(driver, wait);

You can see that code becomes more readable. Also, first group of code contains many lines and you can move it in private method.
I will not add the same comment to other methods, because mechanism in the same